manticore/ethereum.py

```diff
@@ -1423,41 +1423,6 @@ def finalize(self):
         to a STOP or RETURN in the last symbolic transaction).
         """
 
-        # move runnign states to final states list
```

Comment:
what was this section moved to the bottom?

Reply:
Ugh...
The problem of not having source/metadata was at the reporting of "global_XXXX" stuff. Reordering this just made it fail faster for debugging.

Reply:
Update: This section is moved to the bottom so there is at least one state alive from where we can take the runtime bytecode (then assume that is the same in every state) and calculate the coverage over it.